assets/sumologic-windows.yaml

```
@@ -37,7 +37,7 @@ receivers:
 exporters:
   ## Configuration for Sumo Logic Exporter
   ## This exporter supports sending logs, metrics and traces data to Sumo Logic.
-  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/exporter/sumologicexporter
+  ## ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.102.0/exporter/sumologicexporter
```

```suggestion
  ## ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sumologicexporter
```

Let's go with main as we do not update the version in this repo AFAIK